[TASK] Add hint about composer command needing an empty directors (#389)
TYPO3-Documentation/TYPO3CMS-Tutorial-GettingStarted: Documentation/Installation/Install.rst

            # Fetch a basic TYPO3 installation and its dependencies
            ddev composer create "typo3/cms-base-distribution:^13"

..  tip::

>     The commands above expect a completely empty directory. Do not open the project in an

Author: ```suggestion
    The command `composer create-project` expect a completely empty directory. Do not open the project in an
```

>     The following commands expect a completely empty directory. Do not open the project in an

Author: wondering, why do we have composer create for ddev but composer create-project for non-ddev. Didnt notice that before

>     The following commands expect a completely empty directory. Do not open the project in an

Author: ```suggestion
    The commands above expect a completely empty directory. Do not open the project in an
```

>     IDE like phpstorm before they are run. IDEs will usually create a hidden folder like 

Reviewer: ```suggestion
    IDE like PhpStorm before the commands have been executed. IDEs will usually create a hidden folder like 
```

>     :path:`.idea` that will cause an error message with the `composer create-project` command.

Author: ```suggestion
    :path:`.idea` that will cause an error message with the `composer create-project` command. 
    `ddev composer create` also works on non-empty paths.
```

>     :path:`.idea` that will cause an error message with the `composer create-project` command.

Reviewer 2: This is another command than the command from above. What is the difference between "create" and "create-project"

Author: Ack

Author: Strangely the ddev command says composer create and the non-ddev create-project... Do you have any idea which one would be correct?

Reviewer: "create" is AFAIK just a shorthand alias for "create-project". Officially documented is only the latter. We should only use "create-project" then.

Reviewer 2: Please use `create`.
https://ddev.com/blog/getting-started-with-ddev-and-composer/
See section "Built-in Composer commands in DDEV"
`create-project` will execute the original command from Composer and THAT needs a completely empty directory.
`create` is a specific command provided by DDEV and allows creating projects also in non empty environments

Author: So my sentence is correct then, `composer create-project` will fail when executed from an ide that already opened that directory? `create` can only be used with ddev

Reviewer 2: Correct. I just stumbled across the "composer create-project" that suddenly appeared.

Maybe I miss something like:
There is another ddev command called `ddev composer create-project`, which will call the original `composer create-project` command which expects a complete empty directory. Else it will fail. So, please use the `ddev composer create` command as mentioned above, to allow installations in non-empty directories with f.e. `.idea` in case of PhpStorm.

Author: Not everyone is using DDEV and those without don't have the ddev composer create command
